wkconnect/backends/tiff/models.py

```diff
@@ -42,7 +43,7 @@ def to_webknossos(self) -> WkDataSource:
 
     def layers(self) -> List[str]:
         layers = []
-        path = str(self.path())
+        path = str(self.path)
         for filename in listdir(path):
             filepath = Path(filename)
```

Comment:
This should do the same:
```suggestion
        for filepath in self.path.iterdir():
```
The logger statement also needs to use `self.path`. The suggestion doesn't work there, sorry.